annotation/tests/test_post_annotation.py

```diff
@@ -1018,187 +1017,6 @@ def delete_date_fields(annotated_docs: List[dict]) -> None:
-        # rev created, in second upload with the same doc,
-        # func checks, if latest rev was provided, finds out,
-        # that provided rev is not latest,
-        # checks if latest rev and given doc can be merged,
-        # finds out that they have same changed pages
-        # and throws an error)
-        (
-            TASK_ID,
-            DOC_FOR_SECOND_SAVE_BY_USER,
-            1,
-            ASSETS_RESPONSES[0],
-            200,
-            404,
-        ),  # annotated doc was not found
-        (
-            TASK_ID,
-            {
-                "pipeline": PIPELINE_ID,
-                "pages": PAGES,
-            },
-            1,
-            ASSETS_RESPONSES[0],
-            200,
-            400,
-        ),  # user in provided doc should not be None, when saving
-        # annotation by user
-        (
-            TASK_ID,
-            {
-                "user": "48987f26-e0e2-46b6-b674-adaf656fc3a3",
-                "pages": PAGES,
-            },
-            1,
-            ASSETS_RESPONSES[0],
-            200,
-            400,
-        ),  # if user in provided doc does not match user_id
-        # from task
-        (
-            NOT_EXISTING_ID,
-            DOC_FOR_SECOND_SAVE_BY_USER,
-            1,
-            ASSETS_RESPONSES[0],
-            200,
-            404,
-        ),  # task was not found
-        (
-            BAD_ID,
-            DOC_FOR_SECOND_SAVE_BY_USER,
-            1,
-            ASSETS_RESPONSES[0],
-            200,
-            422,
-        ),  # validation error
-        (
-            TASK_ID,
-            {
-                "pages": PAGES,
-            },
-            1,
-            ASSETS_RESPONSES[0],
-            200,
-            422,
-        ),  # fields user and pipeline should not be empty at the same time
-        (
-            TASK_ID,
-            {
-                "user": POST_ANNOTATION_ANNOTATOR.user_id,
-                "pipeline": PIPELINE_ID,
-                "pages": PAGES,
-            },
-            1,
-            ASSETS_RESPONSES[0],
-            200,
-            422,
-        ),  # fields user and pipeline should not be filled at the same time
-        (
-            TASK_ID,
-            DOC_FOR_FIRST_SAVE_BY_USER,
-            1,
-            ASSETS_RESPONSES[0],
-            404,
-            500,
-        ),  # if something wrong with assets
-        (
-            TASK_ID,
-            DOC_FOR_SAVE_NOT_TASK_PAGES,
-            1,
-            ASSETS_RESPONSES[0],
-            200,
-            400,
-        ),  # if pages for save do not belong to task
-        (
-            POST_ANNOTATION_TASK_2["id"],
-            DOC_FOR_FIRST_SAVE_BY_USER,
-            1,
-            ASSETS_RESPONSES[0],
-            200,
-            201,
-        ),
-    ],
-)
-@patch("annotation.annotations.main.KafkaProducer", Mock)
-@responses.activate
-@pytest.mark.skip(reason="tests refactoring")
-def test_post_annotation_by_user_status_codes(
```

Comment:
dont remove this test, remove kafka related stuff. It doesnt test kafka specifically